src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,123 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
+import { getMovie } from '../../api/api';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ movies, addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movieFromApi, setMovieFromApi] = useState('');
+  const [error, setError] = useState(false);
+  const [warn, setWarn] = useState(false);
+
+  const addMovieWithBenefits = () => {
+    setQuery('');
+    setMovieFromApi('');
+    addMovie([
+      ...movies,
+      {
+        title: movieFromApi.Title,
+        description: movieFromApi.Plot,
+        imgUrl: movieFromApi.Poster,
+        imdbUrl: `https://www.imdb.com/title/${movieFromApi.imdbID}/`,
+        imdbId: movieFromApi.imdbID,
+      },
+    ]);
+  };
+
+  return (
+    <>
+      <form
+        className="find-movie"
+        onSubmit={event => event.preventDefault()}
+      >
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={`input${error || warn ? ' is-danger' : ''}`}
+              value={query}
+              onChange={(event) => {
+                setQuery(event.target.value);
+                setError(false);
+                setWarn(false);
+                setMovieFromApi('');
+              }}
+            />
+          </div>
+          {error ? (
+            <p className="help is-danger">
+              Can&apos;t find a movie with such a title
+            </p>
+          ) : null}
```

Comment:
```suggestion
          {error || (
            <p className="help is-danger">
              Can&apos;t find a movie with such a title
            </p>
           )}
```
можна так записати, щоб не використовувати тернарний оператор

Reply:
маєш на увазі && ?

Reply:
Так) error &&  або !error ||. Але з && краще)